lib/bundler/cli.rb

```diff
@@ -151,6 +151,8 @@ def check
         Bundler.rubygems.security_policy_keys.join('|')
     method_option "without", :type => :array, :banner =>
       "Exclude gems that are part of the specified named group."
+    method_option "with", :type => :array, :banner =>
+      "Include gems that are part of the specified named group."
```

Comment:
I thought about that but actually it does not only apply to optional groups. Applying it to a normal group has no effect but also doesn't throw an error, however applying it to a normal group that was previously excluded with `without`, removes it from the excluded groups.